var/spack/repos/builtin/packages/gapfiller/package.py

```diff
@@ -33,7 +33,7 @@ def url_for_version(self, version):
 
     def patch(self):
         with working_dir('.'):
-            files = glob.iglob("*.pl")
+            files = glob.glob("**.pl")
```

Comment:
Is there any reason to switch from iglob to glob?

Reply:
Oops; that was an extraneous change, however the performance won't change much as there are very few files it finds (less than 10 IIRC)

Reply:
I went ahead and reverted the function change.